tool/provision_integration_test_resources.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+if [[ -z "$1" ]]; then
+    echo "Usage: provision_integration_test_resources.sh <CATEGORY>" >&2
+    echo "<CATEGORY> = analytics, auth, storage" >&2
```

Comment:
question: I ran first w auth, then with storage and I noticed it deleted the auth resources (never tried analytics).. User error or bug?

Reply:
You can pass multiple categories e.g. `storage,auth`

Reply:
But is it expected to delete another category if I only specify 1? If so, how would I update or re-deploy a single category? Would I have to specify all unless I really know I only need 1 in the account?

Reply:
Yeah, you need to specify all the categories you want deployed. It will figure out what needs updating, if anything.